src/main/resources/kafka/kafka-topics.json

```diff
@@ -1,5 +0,0 @@
-{
-  "topics": [
-
-  ]
-}
```

Comment:
Any mentions of the file in the documentation for the module?

Reply:
No mentions in any of project files (except in KafkaAdminService's comment that should be removed).